content/assignments/candidate-testing/task-1/index.md

```diff
@@ -70,11 +70,21 @@ To see which tests you passed, scroll up in your terminal.  You will see details
    <img src="../images/Part1-tests-list.png" alt="Results of tests with only task 1 completed" width=50% />
 {{< /rawhtml >}}
 
+[]()
```

Comment:
What is this here for? It is creating extra white space.

Reply:
fixed